integr8ly/integreatly-operator
# MGDAPI-3498 Updating prometheus alert queries

# Issue link
JIRA: https://issues.redhat.com/browse/MGDAPI-3498

# What
Changed prometheus alert queries which many were misfiring. Removed use of flakey `absent` which caused occasional false positives. 

# Verification steps
All of the changes are similar enough so testing one alert effectively tests all changes.

1. Install RHOAM with the changes from this PR.
2. Scale down `threescale-operator-controller-manager` deployment to 0.
3. Scale down `backend-worker` deployment config to 0 and wait 5 minutes.
4. Check prometheus alerts in `Networking` > `Routes` > `Prometheus`
5. 5 minutes after scaling down `backend-worker`, confirm that the `ThreescaleBackendWorkerPod` alert is firing.
6. Scale back up the deployment and deployment-config, wait until the pods launch and check alert stops firing.
